.taskcluster.yml

```diff
@@ -200,7 +200,7 @@ tasks:
                         $merge:
                             - MOBILE_BASE_REPOSITORY: '${baseRepoUrl}'
                               MOBILE_HEAD_REPOSITORY: '${repoUrl}'
-                              MOBILE_HEAD_REF: '${head_branch}'
+                              MOBILE_HEAD_REF: '${short_head_branch}'
```

Comment:
We were having a branch named `refs/head/refs/head/irios-tc-trigger-draft `and so bitrise was failing when cloning as it was not found. With this change that does not happen...

Reply:
oh we many need to undo this change? https://github.com/mozilla-mobile/firefox-ios/pull/18207#issuecomment-1896172112

TBH not sure why we need that suffix in branch name

Reply:
You can probably strip out the `refs/head/` in Taskgraph logic somewhere (e.g in the transform that uses this and forwards it to bitrise)

Reply:
@jjSDET can we look into this please?